cpp/tests/utilities/column_wrapper.hpp

```diff
@@ -205,6 +215,24 @@ class fixed_width_column_wrapper : public detail::column_wrapper {
                 "Unexpected non-fixed width type.");
 
  public:
+  /**
+   * @brief Construct a new fixed width column wrapper by taking ownership of a
+   * column
+   * 
+   * To wrap a pre-existing column, it is required to move it to the constructor
+   * so that it is immediately apparent in the calling code that the old column
+   * is no longer valid
+   * Example:
+   * @code
+   * std::unique_ptr<column> col = make_numeric_column(type, size);
+   * auto wrapped = fixed_width_column_wrapper<dtype>{std::move(col)};
+   * @endcode 
+   * 
+   * @param col The column to take ownership of
+   */
+  fixed_width_column_wrapper(std::unique_ptr<cudf::column>&& col)
```

Comment:
I assume this is only because `to_host` is a member function.

I think it'd be cleaner to make `to_host` a free function that accepts a `column_view`.

Reply:
Yep. So I'm now working on the design requested in #3216. I'm going to have to depend on #3219